pkg/apiserver/profiling/fetcher.go

```diff
@@ -74,6 +77,9 @@ type tiflashFetcher struct {
 }
 
 func (f *tiflashFetcher) fetch(op *fetchOptions) ([]byte, error) {
+	f.client.WithBeforeRequest(func(req *http.Request) {
```

Comment:
This causes protobuf to be stored in the database. However there could be existing svg in the database. I think we should handle this case. Otherwise user will no longer be able to view history profilings.